code/images/SCIENCE.html

```diff
@@ -0,0 +1,165 @@
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">HOME</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">EARTH</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">SPACE</a>
+                </li> 
+              <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">NATURE</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">EVOLUTION</a>
+                </li>
+               
+              <li class="nav-item">
+                  <a class="nav-link" href="http://127.0.0.1:5500/code/index.html#" target="_blank">VIDEOS</a>
+                </li>
+                
+                
+              </ul>
+              <div class="media-icons"> 
+        
+                <i class="fab fa-facebook-f"></i>
+          <i class="fab fa-instagram"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-youtube"></i>
+            </div> 
+
+              
+
+             
+          </nav>
+          <div class="highlights">
+            <p class="highlights1">INTERESTING FACTS</p>
+            <!--The <marquee> HTML element is used to insert a scrolling area of text. -->
+            <marquee direction="left">
+              <p class="highlights2">Hot water freezes faster than cold water. 
+               
+               </p>
+            </marquee>
+          
+          </div>
+ 
+    </header>
+     
+    <main class="main-news">
+      <div class="container1">
+        
+         <img src="./SCIENCE_files/featured.jpg" alt="image" width="900" height="400">
+         <div>
+          <a href="http://127.0.0.1:5500/code/index.html#"><h2 class="featured">FEATURED ARTICLE</h2></a>
+          <p class="featured-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute 
+            irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur 
+            sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore 
+            et dolore magna aliqua. <br><button class="featured-button">Read More.......</button>  </p>
+        </div>
+        </div>
+    </main>
+    
+     <section><div class="news-section">
```

Comment:
I might be wrong here, a rookie you know :D But is it possible to delete the div on row 95 and instead make the <section class="news-section"> I saw [here](https://www.w3schools.com/html/html_blocks.asp) that <section> is a block element aswell. If that works you will have less code doing the same thing. Just a thought :)

Reply:
Ya good suggestion. I will try that